core/src/main/scala/design/hamu/Main.scala

```diff
@@ -12,7 +14,16 @@ object Main {
         .flatMap(job.apply)
       result match {
         case Right(json) => println(json.spaces2)
-        case Left(e) => println(s"Job ${jobName} failed with error ${e.getMessage}")
+        case Left(e) => {
+          e match {
+            case e: FileNotFoundException => println(s"Job ${jobName} failed. No file was found.")
+            case e: NullPointerException =>
```

Comment:
I'm curious on what cases hit `FileNotFoundException` and `NullPointerException` since cases seem to be the same? No file was found?

Reply:
I actually couldn't get it to trigger a file not found exception; I think it goes straight to a null pointers so I'll remove that file not found